src/Component/Metadata/Operation.php

```diff
@@ -13,11 +13,13 @@
 
 namespace Sylius\Component\Resource\Metadata;
 
-class Operation
+/**
+ * @experimental
+ */
+abstract class Operation
 {
     public function __construct(
         protected ?string $name = null,
-        protected ?array $methods = null,
         protected ?string $path = null,
         protected ?string $routePrefix = null,
```

Comment:
If we want to separate the HTTP operation from generic operations do we need these two lines? I'm wondering if we can see a direct usage of it